### PR TITLE
Adding explanation about cURL on Windows

### PR DESCRIPTION
**Pull request by marcellasiqueira**
#### What is the purpose of this pull request?

This pull request aims to explain the installation of cURL in case of using Windows.

#### What problem is this solving?

The documentation was unclear regarding the fact that users of older versions of Windows have to install cURL in order to run the given commands.

#### Types of changes

- [x] Content fix
- [ ] Answer sheet fix
- [ ] Course refactor
- [ ] New course :rocket:
- [ ] Script bug fix
- [ ] Script feature

### CODE REVIEW

**Review by fabianaferreira — APPROVED** [review]
LGTM

File: courses/service-course/steps/06_clients-masterdata/en.md [hunk 1]
```diff
@@ -204,6 +204,10 @@ Now you are good to go!
    --header 'Content-Type: application/json'
    ```
 
+    > **Attention**: If you have a version of Windows earlier than Windows 10, version 1803, it’s necessary to [download and install _cURL_](https://curl.se/windows/). Otherwise, it is installed by default.
+    >
+    > To run the command _cURL_ on **Windows**, it's necessary to substitute single quotes (`'`) for double quotes (`"`).
```
Comment (PedroAntunesCosta) [inline courses/service-course/steps/06_clients-masterdata/en.md:209]: cURL aqui é o comando em si ou o tipo do comando? Se for o comando, recomendo trecho de código (``), se for cURL no sentido amplo recomendo negrito (****).

File: courses/service-course/steps/06_clients-masterdata/pt.md [hunk 2]
```diff
@@ -208,7 +208,9 @@ Agora você está pronto para começar!
    --header 'Content-Type: application/json'
    ```
 
-   > **Atenção**: Para rodar o comando _cURL_ no **Windows**, é necessário substituir as aspas simples (`'`) por aspas duplas (`"`).
+    > **Atenção**: Se você tem uma versão do Windows anterior ao Windows 10, versão 1803, é necessário [baixar e instalar _cURL_](https://curl.se/windows/). Caso contrário, ele já é instalado por default.
+    >
+    > Para rodar o comando _cURL_ no **Windows**, é necessário substituir as aspas simples (`'`) por aspas duplas (`"`).
```
Comment (PedroAntunesCosta) [inline courses/service-course/steps/06_clients-masterdata/pt.md:213]: cURL aqui é o comando em si ou o tipo do comando? Se for o comando, recomendo trecho de código (``), se for cURL no sentido amplo recomendo negrito (****).

**Review by PedroAntunesCosta — CHANGES_REQUESTED** [review]

**Review by PedroAntunesCosta — APPROVED** [review]
LGTM

**Pull request merged**